core/src/com/cloud/agent/resource/virtualnetwork/VirtualRoutingResource.java

```diff
@@ -272,7 +272,10 @@ public boolean configure(final String name, final Map<String, Object> params) th
         _port = NumbersUtil.parseInt(value, 3922);
 
         value = (String)params.get("router.aggregation.command.each.timeout");
-        _eachTimeout = Duration.standardSeconds(NumbersUtil.parseInt(value, 10));
+        _eachTimeout = Duration.standardSeconds(NumbersUtil.parseInt(value, (int)VRScripts.VR_SCRIPT_EXEC_TIMEOUT.getStandardSeconds()));
```

Comment:
@borisstoyanov however this fix makes sense to use the default value when not parse-able.